src/transformState.js

```diff
@@ -4,8 +4,32 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
+      case action.type === 'addProperties':
```

Comment:
use proper `switch` case, put your type as a value and it's values for cases, like this:
```suggestion
    switch (action.type) {
      case 'addProperties':
```